book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] According to the project associations, kindly make sure `Getters` and `Setters` are present for `@rentals` i.e `:rentals` should be also added on the `attr_accessor`. For more details, you can read **[this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)**.